packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -56,7 +55,7 @@ class Layout extends React.Component {
             onRequestClose={this.toggleNav}
           />
           <main id="main-container" className={cx('main-container', this.state.isMobileNavOpen && 'nav-open')}>
-            <button className={cx('nav-burger')} onClick={this.toggleNav}><IconMenu /></button>
+            <Nav.Burger className={cx('nav-burger')} handleClick={this.toggleNav} />
```

Comment:
haha can we call this toggler instead

Reply:
You can leave it if we can call something fries?

Reply:
haha i really liked typing out nav burger the whole time, I vote we find something to call fries

Reply:
+1![burger](http://smokeybones.com/wp-content/uploads/2015/11/smokehouse-burger.jpg)